tests/pytests/test_data_naif.py

```diff
@@ -2,41 +2,39 @@
 
 import numpy as np
 
-from ale.base import data_naif
+from unittest.mock import patch
 
-# 'Mock' the spice module where it is imported
-from conftest import SimpleSpice
-
-simplespice = SimpleSpice()
-data_naif.spice = simplespice
+from ale.base.data_naif import NaifSpice
 
 @pytest.fixture
 def test_naif_data():
-    naif_data = data_naif.NaifSpice()
+    naif_data = NaifSpice()
     naif_data.instrument_id = "INSTRUMENT"
     naif_data.target_name = "TARGET"
     naif_data.ephemeris_time = [0, 1]
 
     return naif_data
 
 def test_target_id(test_naif_data):
-    assert test_naif_data.target_id == -12345
+    with patch('spiceypy.bods2c', return_value=-12345) as bods2c:
+        assert test_naif_data.target_id == -12345
+        bods2c.assert_called_once_with("TARGET")
 
 def test_pixel_size(test_naif_data):
-    assert test_naif_data.pixel_size == (0.001)
+    with patch('spiceypy.bods2c', return_value=-12345) as bods2c, \
+         patch('spiceypy.gdpool', return_value=[1]) as gdpool:
+        assert test_naif_data.pixel_size == (0.001)
+        bods2c.assert_called_once_with("INSTRUMENT")
+        gdpool.assert_called_once_with("INS-12345_PIXEL_SIZE", 0, 1)
 
 def test_radii(test_naif_data):
-    np.testing.assert_equal(test_naif_data.target_body_radii, np.ones(3))
-
-def test_naif_keywords(test_naif_data):
-    np.testing.assert_equal(test_naif_data.isis_naif_keywords['BODY-12345_RADII'], np.ones(3))
-    np.testing.assert_equal(test_naif_data.isis_naif_keywords['BODY_FRAME_CODE'], 2000)
-    np.testing.assert_equal(test_naif_data.isis_naif_keywords['INS-12345_PIXEL_SIZE'], (0.001))
-    np.testing.assert_equal(test_naif_data.isis_naif_keywords['INS-12345_ITRANSL'], np.ones(3))
-    np.testing.assert_equal(test_naif_data.isis_naif_keywords['INS-12345_ITRANSS'], np.ones(3))
-    np.testing.assert_equal(test_naif_data.isis_naif_keywords['INS-12345_FOCAL_LENGTH'], np.ones(1))
-    np.testing.assert_equal(test_naif_data.isis_naif_keywords['INS-12345_BORESIGHT_LINE'], np.ones(1))
-    np.testing.assert_equal(test_naif_data.isis_naif_keywords['INS-12345_BORESIGHT_SAMPLE'], np.ones(1))
```

Comment:
Removed because mocking all of this out properly is going to couple the test to the code itself so tightly that there's almost no point in it.